Source/DataSources/ModelVisualizer.js

```diff
@@ -176,6 +179,15 @@ define([
                     modelData.animationsRunning = runAnimations;
                 }
 
+                if (modelData.animationsRunning && modelData.multiplier !== multiplier) {
+                    var animationLength = model.activeAnimations.length;
```

Comment:
What exactly is an `activeAnimation` can an animation become inactive?  And if that happens will it's multiplier setting become out of date?